.github/workflows/container-build.yml

```diff
@@ -59,5 +60,21 @@ jobs:
       - name: creds
         run: gcloud auth configure-docker --quiet
 
+      - name: Formatted labels
+        id: labels
+        run: |
+          FORMATED_LABELS="--image-label commit-hash=$GITHUB_SHA"
+
+          BRANCH_NUMBER=$(gh pr list --state all --search "sha:$GITHUB_SHA" --label "breaking-change" | awk '{print $1}')
+          echo "Branch Number: $BRANCH_NUMBER" 
+
+          # Check if a pull request number was found
+          if [ -n "$BRANCH_NUMBER" ]; then
+            FORMATED_LABELS+=" --image-label breaking-change=true"
+          fi
+          echo "FORMATED_LABELS='$FORMATED_LABELS'" >> $GITHUB_OUTPUT
```

Comment:
need to set the env var

```
GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
check your fork execution: https://github.com/javanlacerda/fulcio/actions/runs/11242081243/job/31254992474#step:7:16

Reply:
My fork execution failed because I removed the steps that setup the gcloud SDK, as I just wanted to test that the `ko` command was being created correctly, without uploading the images.  Here you can check the how my action definition looks like: https://github.com/javanlacerda/fulcio/commit/1c8adf3ac29767bfe91a0f1ab334766743c83638

Reply:
You're right! adding here

Reply:
done, thanks!!